extensions/emoji/js/src/admin/index.js

```diff
@@ -0,0 +1,19 @@
+import app from 'flarum/admin/app';
+import {version} from "../forum/cdn";
```

Comment:
since this is needed from both `forum` and `admin` could you please move it to `common` and update import paths?